active-directory-b2c-wpf/MainWindow.xaml.cs

```diff
@@ -101,10 +93,10 @@ private async void CallApiButton_Click(object sender, RoutedEventArgs e)
         {
             AuthenticationResult authResult = null;
             var app = App.PublicClientApp;
-            IEnumerable<IAccount> accounts = await App.PublicClientApp.GetAccountsAsync();
+            var accounts = await app.GetAccountsAsync(App.PolicySignUpSignIn);
```

Comment:
The only difference is the extra policy (in addition to the authority, of course)